tests/test_runs/test_run_functions.py

```diff
@@ -385,6 +413,40 @@ def test_initialize_cv_from_run(self):
         self.assertEquals(modelS.cv.random_state, 62501)
         self.assertEqual(modelR.cv.random_state, 62501)
 
+    def test_get_run_metric_score(self):
+
+        # construct sci-kit learn classifier
+        clf = Pipeline(steps=[('imputer', Imputer(strategy='median')), ('estimator', RandomForestClassifier())])
+
+
+        # download task
+        task = openml.tasks.get_task(7)
+
+        # invoke OpenML run
+        run = openml.runs.run_model_on_task(task, clf)
+
+        # compare with the scores in user defined measures
+        accuracy_scores_provided = []
+        for rep in run.fold_evaluations['predictive_accuracy'].keys():
+            for fold in run.fold_evaluations['predictive_accuracy'][rep].keys():
+                accuracy_scores_provided.append(run.fold_evaluations['predictive_accuracy'][rep][fold])
+        accuracy_scores = run.get_metric_score(sklearn.metrics.accuracy_score)
+        self.assertEquals(sum(accuracy_scores_provided), sum(accuracy_scores))
```

Comment:
You could use [numpy-specific test-functions](https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_almost_equal.html#numpy.testing.assert_array_almost_equal) here to make the code more readable.